frontend/src/scenes/funnels/FunnelBarChart/FunnelBarChart.tsx

```diff
@@ -114,14 +114,14 @@ export function FunnelBarChartComponent({
                             <td key={stepIndex}>
                                 {isUsingDataExploration ? (
                                     <StepLegendDataExploration
-                                        step={step.nested_breakdown?.length ? step.nested_breakdown[0] : step}
+                                        step={step}
```

Comment:
wow this is clever!

I'm _slightly_ scared this doesn't work for other existing use cases, but not sure, basic ones seem to all work

Reply:
it should mirror how the logic is handled on horiztonal bar charts. (those don't have the same bug)

Reply:
I hope this won't break anything but I think it might but let's see xD (why did we have step.nested_breakdown[0] originally ? )

Reply:
I would assume for a funnel like this https://app.posthog.com/insights/IlZLZRIH it might break but I could be wrong

Reply:
This PR is literally a pure revert of this PR: #10750 – so I guess this might break the persons modal. Can you give this a check?

Reply:
Awesome, seems to be correct. Let's replace this Chesterton's fence